src/Microsoft.Health.Fhir.Core.UnitTests/Models/PartialDateTimeTests.cs

```diff
@@ -14,172 +14,133 @@ namespace Microsoft.Health.Fhir.Core.UnitTests.Models
 {
     public class PartialDateTimeTests : IDisposable
```

Comment:
The big change made to this file was creating `PartialDateTime` objects by passing in a `datetime` string to the `PartialDateTime.Parse` method. This is the code path that is follow in a non-test environment.

For the most part, test cases remain the same.